libraries/javascript/eddystone-advertising/README.md

```diff
@@ -0,0 +1,240 @@
+# Eddystone Advertising Library
+
+The Eddystone Advertising Library abstracts low level concepts of the Eddystone
+protocol and wraps existing Advertising APIs so that developers don't have to
+worry about these low level details. The library exposes simple functions that
+developers can use to advertise a Valid Eddystone packet from their device.
+
+** NOTE ** Currently only ChromeOS is supported.
```

Comment:
okay, looks like there is preliminary groundwork for the other frame types but indeed only URL is supported.